jax/_src/numpy/lax_numpy.py

```diff
@@ -1887,6 +1887,12 @@ def stack(arrays: np.ndarray | Array | Sequence[ArrayLike],
       new_arrays.append(expand_dims(a, axis))
     return concatenate(new_arrays, axis=axis, dtype=dtype)
 
+@util.implements(getattr(np, 'unstack', None))
+@partial(jit, static_argnames="axis")
+def unstack(x: np.ndarray | Array, /, *, axis: int = 0) -> tuple[Array, ...]:
```

Comment:
We typically use `x: ArrayLike` for numpy API inputs.

If scalar inputs are not allowed, we should do that as a runtime check here (rather than letting the error come from within the implementation).